### PR TITLE
qogir-theme: init at 2018-11-12

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

Add [qogir-theme](https://github.com/vinceliuice/Qogir-theme)

>[Qogir](https://www.gnome-look.org/p/1230631/) is a flat Design theme for GTK 3, GTK 2 and Gnome-Shell which supports GTK 3 and GTK 2 based desktop environments like Gnome, Unity, Budgie, Pantheon, XFCE, Mate, etc.

qogir

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by xeji** [comment]
@GrahamcOfBorg eval

File: pkgs/data/themes/qogir/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, gdk_pixbuf, librsvg, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "qogir-${version}";
```
Comment (worldofpeace) [inline pkgs/data/themes/qogir/default.nix]: Why not?
```suggestion
  pname = "qogir-theme";
```

File: pkgs/data/themes/qogir/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  version = "2018-11-12";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = "qogir-theme";
```
Comment (worldofpeace) [inline pkgs/data/themes/qogir/default.nix]:
```suggestion
    repo = pname;
```
https://github.com/NixOS/nixpkgs/pull/52918/files#r244081530

File: pkgs/data/themes/qogir/default.nix (hunk 1 above)
Comment (romildo) [reply pkgs/data/themes/qogir/default.nix]: Done.

File: pkgs/data/themes/qogir/default.nix (hunk 2 above)
Comment (romildo) [reply pkgs/data/themes/qogir/default.nix]: Done.

File: pkgs/data/themes/qogir/default.nix (hunk 1 above)
Comment (JohnAZoidberg) [reply pkgs/data/themes/qogir/default.nix]: Question: Is `pname` and version automatically combined into the name with version? I've only ever seen `pname` in Python packages.

Comment (romildo) [reply pkgs/data/themes/qogir/default.nix]: Yes, with `buildGoPackage`, `stdenv.mkDerivation` and Python packages.

**Comment by worldofpeace** [comment]
@romildo Thanks again :+1:

**Pull request merged**